build.sh

```diff
@@ -295,6 +296,10 @@ while :; do
         verbose)
             __VerboseBuild=1
             ;;
+	staticcurl)
+            staticCurl=" -DCMAKE_STATIC_CURL=1"
+            __CMakeExtraArgs="$__CMakeExtraArgs$staticCurl"
```

Comment:
updated. I did that because it didn't work before, I expect I was doing something wrong.